Hidden-black.md

```diff
@@ -0,0 +1,51 @@
+---
+name: "@Hidden-black"
+project: "Toy Car"
+
+---
+
+## Summary
+This would be my first step towards using microcontrollers and i am super exited to do it.
+I would be building multiple projects with the use of Arduino. I always wanted to build projects with the use of miro-controllers but never got a chance to build one.
+
+## Plan
+
+I first objective with this project will be to create a simple project like a motion sensor that detects when someone is coming near my room. I would also try to build diffrent things like a gas sensor before I get started with the main project to acquire the basic knowledge of Arduino.
+<br>
+The small toy car would help me with learning programing like controling all 4 motors seprately using motor driver
+
+### 1. Get The materials
+- Get everything required for this project along with other everyday thing (like cardboard)
+
+### 2. Build a frame for toy car
+- Plan the car in blender using simple shapes like cuboid and celender to get it proper placement and dimmensions of the components
+- Build the frame 
```

Comment:
@Hidden-black thanks for pushing the changes! how do you plan on building the frame?

also, is there a guide that you plan on referencing?